src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -1633,6 +1633,12 @@ AuthorizedAuthenticationState doAuthorization(final boolean requireLoginPermissi
             final AuthorizationIdentity authorizationIdentity = realmIdentity.getAuthorizationIdentity();
             final SecurityDomain domain = capturedIdentity.getSecurityDomain();
 
+            if (log.isDebugEnabled()) {
```

Comment:
The guideline is, any logging that happens on a per-request basis should be TRACE.  This is old but the section on levels is still relevant: https://developer.jboss.org/wiki/LoggingStandards